.travis.yml

```diff
@@ -34,6 +34,12 @@ install: true
 
 jobs:
   include:
+    - stage: "Verify vendor"
+      script: bash build/tools/verifyVendor.sh
+      name: "check if the vendor is up to date"
+    - script: bash build/tools/verifyDepVersion.sh
+      name: "check if patches in dependencies are present"
```

Comment:
Thanks @subpathdev for quick fix! And can we move this to `Makefile` and replace `dep check` in it :)

Reply:
Yes of course